docs/writing-steps.rst

```diff
@@ -0,0 +1,56 @@
+.. _good-bdd:
```

Comment:
It could, but it feels a lot more relevant here where we hand the user the box of axes by which she can cause so much havoc. Perhaps Aloe needs a more complete essay on good BDD practices?